tests/main/interfaces-location-control/task.yaml

```diff
@@ -28,9 +28,12 @@ prepare: |
     tests.session -u test prepare
 
 restore: |
-    systemctl stop test-snapd-location-control-provider.service
+    snap remove test-snapd-location-control-provider
     tests.session -u test restore
 
+    # Stop any scope which could be still running on the user session
+    systemctl stop --state=running --type scope --user snap.*
```

Comment:
this sort of feels like a bug in the test if we have to do this manually, no?

could you at least add a TODO of some sort for each of the tests where we have to do this, i.e. where we have to manually kill all the scopes left over?

Reply:
sure